sale_order_color_code_tintometric_system_stock/views/stock_picking_views.xml

```diff
@@ -0,0 +1,12 @@
+<odoo>
+    <record id="stock_picking_view_form" model="ir.ui.view">
+        <field name="name">stock.picking.view.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//tree//field[@name='description_picking']" position="after">
+                <field name="color_code_tintometric" readonly="1"/>
```

Comment:
Add `optional="hide"`. And, if a `form` view is also available, add the field there too. And `readonly` is actually unnecessary here, since field is `readonly` itself.